config/routes.rb

```diff
@@ -10,6 +10,8 @@
       resources :notes, only: [ :create, :update ]
       resources :external_pledges, only: [ :create, :update, :destroy ]
     end
+    get 'get_clinics', to: 'patients#get_clinics', as: 'get_clinics'
```

Comment:
Per above, I think these might be better suited for the clinics controller. What are your thoughts?